src/Messages/OWSMessageSender.m

```diff
@@ -680,22 +721,29 @@ - (void)groupSend:(NSArray<SignalRecipient *> *)recipients
     }];
 
     [completionFuture catchDo:^(id failure) {
-        // failure from toc_thenAll yeilds an array of failed Futures, rather than the future's failure.
+        // failure from toc_thenAll yields an array of failed Futures, rather than the future's failure.
         if ([failure isKindOfClass:[NSArray class]]) {
             NSArray *groupSendFutures = (NSArray *)failure;
             for (TOCFuture *groupSendFuture in groupSendFutures) {
                 if (groupSendFuture.hasFailed) {
                     id failureResult = groupSendFuture.forceGetFailure;
                     if ([failureResult isKindOfClass:[NSError class]]) {
-                        return failureHandler(failureResult, YES);
+                        NSError *error = failureResult;
+                        // Some errors should be ignored when sending messages
+                        // to groups.  See discussion on
+                        // NSError (OWSMessageSender) category.
+                        if ([error shouldBeIgnoredForGroups]) {
```

Comment:
...But there are some errors that we want to ignore when send messages to groups.